components/infobox/wikis/rainbowsix/infobox_league_custom.lua

```diff
@@ -0,0 +1,273 @@
+---
+-- @Liquipedia
+-- wiki=rainbowsix
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Tier = require('Module:Tier')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+local PageLink = require('Module:Page')
+local PrizePoolCurrency = require('Module:Prize pool currency')
```

Comment:
for follow up:
this module should probably be added to the git (after cleanup and standardizing)

Reply:
This, or made part of Infobox/League